lib/puppet/error.rb

```diff
@@ -21,7 +21,7 @@ module ExternalFileError
     # 4 args including the position on the line.
     #
     def initialize(message, file = nil, line = nil, pos = nil, original = nil)
-      if pos.kind_of? Exception
+      if pos.is_a? Exception
```

Comment:
Previously we matched any Exception or subclass, but now we only match an Exception, which I don't think we want.